fe/lib/components/LocationSuggest/LocationSuggest.tsx

```diff
@@ -50,12 +54,25 @@ export const LocationSuggest = forwardRef<HTMLInputElement, Props>(
       fetchPolicy: 'no-cache',
     });
 
+    const [
+      nearestLocations,
+      {
+        data: nearestLocationsData,
+        error: nearestLocationsError,
+        loading: nearestLocationsLoading,
+      },
+    ] = useLazyQuery(NEAREST_LOCATIONS, {
```

Comment:
Not for this PR, but it would be good to set up [TypeScript operation codegen](https://graphql-code-generator.com/docs/plugins/typescript-operations) so we can strongly type this like `useLazyQuery<NearestLocationsQuery, NearestLocationsQueryVariables>(...)`

Reply:
I think these types are already available for us to use. 

I noticed this as well and some mocks also not being typed. I think it'll be good to clean them up in a follow-up.